konnectivity-socks5-proxy/main.go

```diff
@@ -70,16 +71,19 @@ func NewStartCommand() *cobra.Command {
 			panic(err)
 		}
 
-		dialFunc := dialFunc(caCertPath, clientCertPath, clientKeyPath, proxyHostname, proxyPort, connectDirectlyToCloudAPIs)
+		fallbackToManagementCluster := new(bool)
```

Comment:
This variable is going to be accessed by different go routines potentially all at the same time. It should at least be protected by a RWMutex.

Reply:
Added the RWMutex